### DISCUSSION TITLE
Problem getting kitty to be transparent

### DISCUSSION THREAD

**Discussion by Asmenys** [kovidgoyal/kitty#3901] [Q&A] [answered]
Greetings, I've been trying to set up kitty and have been unable to get it to be transparent. 
I have enabled _dynamic_background_opacity_ and have played around with _background_opacity_ to no avail. My question is: what am I missing?
I am using arch linux with bspwm as my window manager.

**Comment by kovidgoyal** [comment, marked as answer]
See the second sentence here: https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.background_opacity

**Reply by Asmenys** [reply]
Thank you, the problem was that I forgot to add picom to bswpmrc which resulted in the compositor never starting, my bad.
